ark/utils/misc_utils.py

```diff
@@ -137,15 +163,22 @@ def verify_in_list(**kwargs):
     test_list, good_values = kwargs.values()
 
     if not np.isin(test_list, good_values).all():
-        bad_vals = ','.join([str(val) for val in test_list if val not in good_values])
         test_list_name, good_values_name = kwargs.keys()
-        test_list_name = test_list_name.replace('_', ' ')
-        good_values_name = good_values_name.replace('_', ' ')
+        test_list_name = test_list_name.replace("_", " ")
+        good_values_name = good_values_name.replace("_", " ")
 
-        err_str = ("Invalid value(s) provided for %s variable: value(s) %s not found"
-                   " in %s list")
+        # Calculate the difference between the `test_list` and the `good_values`
+        difference = [str(val) for val in test_list if val not in good_values]
 
-        raise ValueError(err_str % (test_list_name, bad_vals, good_values_name))
+        # Only printing up to the first 10 invalid values.
+        err_str_1 = ("Displaying {0} of {1} invalid value(s) provided for list {2:^}.\n").format(
+            min(len(difference), 10), len(difference), test_list_name
+        )
+
+        err_str_3 = create_invalid_data_str(difference)
+
+        err_str = err_str_1 + err_str_3
```

Comment:
The names `err_str_1` and `err_str_3` should be more descriptive. You could rename each one of these, but I personally prefer to just have one `err_str` and append each subsequent update:

```
err_str = ("Displaying {0} of {1} invalid value(s) provided for list {2:^}.\n").format(
    min(len(difference), 10), len(difference), test_list_name
)

err_str += create_invalid_data_str(difference)
```